Src/Newtonsoft.Json/Linq/JToken.cs

```diff
@@ -334,7 +334,7 @@ public IEnumerable<JToken> BeforeSelf()
         /// <typeparam name="T">The type to convert the token to.</typeparam>
         /// <param name="key">The token key.</param>
         /// <returns>The converted token value.</returns>
-        public virtual T Value<T>(object key)
+        public virtual T? Value<T>(object key)
```

Comment:
Is adding nullability annotations to generic type arguments now supported in C# 9? I didn't see anything related to that in the blog posts or in the what's new in C# 9 guide at https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-9.

Reply:
This need backward compatibility, need use directives #nullable and #if NET50?

Reply:
> Is adding nullability annotations to generic type arguments now supported in C# 9? I didn't see anything related to that in the blog posts or in the what's new in C# 9 guide at [docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-9](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-9).

https://github.com/dotnet/csharplang/blob/9807f9b2e2105255c0a658f3afc908769bb31339/proposals/csharp-9.0/unconstrained-type-parameter-annotations.md